dll/win32/iernonce/iernonce.cpp

```diff
@@ -34,7 +38,12 @@ DllMain(HINSTANCE hinstDLL,
    return TRUE;
 }
 
-VOID WINAPI RunOnceExProcess(HWND hwnd, HINSTANCE hInst, LPCSTR path, int nShow)
+extern "C" VOID WINAPI
```

Comment:
Is the extern needed here?

Reply:
It is, if this function is exported (it needs a C decoration)